docs/resources/azurerm_resource_groups.md

```diff
@@ -95,13 +95,23 @@ is a string value.
 
 ## Attributes
 
+  - `ids`
   - `names`
+  - `tags`
 
 ### names
 
+The names property provides a list of all the Resource Group ids.
```

Comment:
Isn't it supposed to be: `The ids property ...`

Reply:
good catch I'll add a follow-up PR